sendgrid-php.php

```diff
@@ -1,3 +1,28 @@
 <?php
-require __DIR__ . '/vendor/autoload.php';
-?>
\ No newline at end of file
+/**
+ * This file is used to load the Composer autoloader if required.
+ */
+
+use SendGrid\Mail\Mail;
+
+// Define path/existence of Composer autoloader
+$composerAutoloadFile = __DIR__ . '/vendor/autoload.php';
+$composerAutoloadFileExists = (is_file($composerAutoloadFile));
+
+// Can't locate SendGrid\Mail\Mail class?
+if (!class_exists(Mail::class)) {
+    // Suggest to load Composer autoloader of project
+    if (!$composerAutoloadFileExists) {
+        //  Can't load the Composer autoloader in this project folder
+        error_log("Composer autoloader not found. Execute 'composer install' in the project folder first.");
+    } else {
+        // Load Composer autoloader
+        require_once $composerAutoloadFile;
+
+        // If desired class still not existing
+        if (!class_exists(Mail::class)) {
+            // Suggest to review the Composer autoloader settings
+            error_log("Error finding SendGrid classes. Please review your autoloading configuration.");
+        }
+    }
+}
```

Comment:
I saw this PR https://github.com/sendgrid/sendgrid-php/pull/984 before this one. Now, after reviewing this one, I think we should merge this first and then revisit the wording in #984.

Reply:
Exactly. This PR gives the base to optimize #984, but without it this PR is useless to implement (chicken or the egg).